support/cas-server-support-hazelcast-core/src/main/java/org/apereo/cas/hz/HazelcastConfigurationFactory.java

```diff
@@ -126,6 +126,12 @@ public static Config build(final BaseHazelcastProperties hz) {
             networkConfig.setPublicAddress(cluster.getPublicAddress());
         }
 
+        if (!org.springframework.util.CollectionUtils.isEmpty(cluster.getOutboundPorts())) {
```

Comment:
Conditional should unnecessary with an empty list above.

Reply:
yes